contrib/generate_precompile.jl

```diff
@@ -425,8 +435,11 @@ generate_precompile_statements() = try # Make sure `ansi_enablecursor` is printe
             ps.head = :tuple
             # println(ps)
             ps = Core.eval(PrecompileStagingArea, ps)
-            precompile(ps...)
-            n_succeeded += 1
+            if precompile(ps...)
+                n_succeeded += 1
+            else
+                @warn "Failed to precompile expression" form=statement _module=nothing _file=nothing _line=0
```

Comment:
How much noise does this generate?

Reply:
0 failures locally so none currently. That's why I was thinking it would be good to have it be noisy

Reply:
LGTM